src/wallet/rpcwallet.cpp

```diff
@@ -768,8 +770,10 @@ UniValue getbalance(const JSONRPCRequest& request)
 
     LOCK2(cs_main, pwallet->cs_wallet);
 
-    if (request.params.size() == 0)
-        return  ValueFromAmount(pwallet->GetBalance());
+    bool ignoreUnspendable = request.params[3].isNull() || !request.params[3].get_bool();
+    if ((request.params[0].isNull() || (request.params[0].get_str() == "" && !ignoreUnspendable)) && request.params[1].isNull() && request.params[2].isNull()) {
```

Comment:
I find it surprising that `minconf` and `watchonly` only have effect if account is specified, and now `include_unspendable` only has effect if account is not specified. There doesn't appear to be any user documentation explaining that.

I'm afraid I don't know the history or motivation for that. @ryanofsky made changes in this area recently. Perhaps he can shed light on this.